classes/UpgradeSelfCheck.php

```diff
@@ -322,10 +324,28 @@ public function getRequirementWording(int $requirement, bool $isWebVersion = fal
                     'message' => $this->translator->trans('We were unable to check your PHP compatibility with the destination PrestaShop version.'),
                 ];
 
-            case self::TEMPERED_FILES_LIST_NOT_EMPTY:
+            case self::CORE_TEMPERED_FILES_LIST_NOT_EMPTY:
+                if ($isWebVersion) {
+                    // TODO the link must be integrated when implementing the modal in the web part
+                    $message = $this->translator->trans('Some core files have been altered, customization made on these files will be lost during the update. <a class="link" href="">See the list of alterations.</a>');
```

Comment:
for me "See the list of alterations." nothing to do on the CLI version, this clearly refers to a button allowing you to display a modal

Reply:
In addition on the CLI version we directly display the list so a "See the list of alterations." is redundant since we have it before our eyes

Reply:
Agreed. My comment was more about the tags [1] and [/1] and copy pasted your string in the previous example, sorry for the confusion.

Reply:
So I still don't understand where you were going with this.

Reply:
Replace the html by `[1]` and `[/1]` in the contents sent to the translator, there is no point in translating this on Crowdin.
Even though it could be done in a PR dedicated to the web, we will send the proper string to the translation files now.